recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
I can see how calling the formula `ampl-mp` here and in Homebrew would be consistent with the Github repository, but if it's going to be called ASL in some places and not in others, I think we'll all be worse off.